manager/orchestrator/replicatedjob/orchestrator.go

```diff
@@ -47,174 +49,44 @@ func (o *Orchestrator) Run(ctx context.Context) {
 	// method, and so should be the absolute first thing we defer.
 	defer close(o.doneChan)
 
-	// TODO(dperny): this will be a case in the main select loop, but for now
-	// just block until stopChan is closed.
-	<-o.stopChan
-}
-
-// reconcileService reconciles the replicated job service with the given ID by
-// checking to see if new replicas should be created. reconcileService returns
-// an error if there is some case prevent it from correctly reconciling the
-// service.
-func (o *Orchestrator) reconcileService(id string) error {
 	var (
-		service *api.Service
-		tasks   []*api.Task
-		viewErr error
+		services []*api.Service
 	)
-	// first, get the service and all of its tasks
-	o.store.View(func(tx store.ReadTx) {
-		service = store.GetService(tx, id)
 
-		tasks, viewErr = store.FindTasks(tx, store.ByServiceID(id))
+	o.store.View(func(tx store.ReadTx) {
+		// TODO(dperny): figure out what to do about the error return value
+		// from FindServices
```

Comment:
For now let's at least log it?

Reply:
I don't have any logging added yet because I haven't figured out how to test it yet, so I've deferred adding logging.